meta-lmp-bsp/wic/sdimage-imx8-spl-sota.wks.in

```diff
@@ -0,0 +1,22 @@
+# short-description: Create SD card image with a boot partition
+# long-description:
+# Create an image that can be written onto a SD card using dd for use
+# with i.MX 8 SoC family
+# SPL binary is just renamed imx-boot (contains: u-boot-spl-nodtb.bin; dt-spl.dtb; ddr firmware binaries)
+# u-boot partition contains rawcopy of U-Boot FIT image (contains: bl31.bin; tee.bin; u-boot.bin; u-boot.dtb)
+#
+# The disk layout used is:
+#  - ---------- -------------- --------------------
+# | | SPL  | u-boot  |     boot     |    rootfs   |
+#  - ---------- -------------- --------------------
+# ^ ^      ^         ^              ^
+# | |      |         |              |
+# 0 |      384KiB    4MiB           68MiB + rootfs + IMAGE_EXTRA_SPACE (default 10MiB)
+#   ${IMX_BOOT_SEEK} 32 or 33kiB, see reference manual
+#
+part SPL --source rawcopy --sourceparams="file=imx-boot" --ondisk mmcblk --no-table --align ${IMX_BOOT_SEEK}
```

Comment:
iirc, we decided to keep copies of `imx-boot` and `u-boot.itb` images so we're aligned with SD setup, correct?

Reply:
Yes.  We want to keep these here so that SD card can be supported.